modular-docs-manual/content/topics/module_guidelines-concept.adoc

```diff
@@ -6,7 +6,7 @@ A concept module requires a short introduction and optionally, can also include
 
 When planning a concept module, look at nouns in related procedure modules and assemblies to find the concepts to explain to users.
 Explain only things that are visible to users.
-Even if a concept is interesting, it probably does not require explanation if it is not visible to users.
+Even if a concept is interesting, it probably does not require an explanation if it is not visible to users.
```

Comment:
"an" is not necessary here, so in the interest of minimalism (use as few words as possible), I would not add it

Reply:
@jenmalloy Thank you for the feedback - I've added the missing article (above). However, I still believe that "an" fits here. I'm not a native speaker though.